pkg/cmd/space/view/view.go

```diff
@@ -44,7 +44,7 @@ func NewCmdView(f factory.Factory) *cobra.Command {
 			}
 
 			opts.Client = client
-			opts.Host = os.Getenv("OCTOPUS_HOST")
+			opts.Host = os.Getenv(constants.EnvOctopusHost)
```

Comment:
I realise this is not your code @benPearce1 and is leftover from some earlier CLI code, no fault of yours.
Nevertheless this is a bug.

The `Host` can come from three places (lowest priority first):
- Config File
- Environment
- Command Line

And this only uses env. IIRC either the Client or the factory has a GetHost method on it that you can use for this

Reply:
`Factory.GetCurrentHost()` looks like it will do the trick 👍